config/settings/grafana.q

```diff
@@ -0,0 +1,7 @@
+/ Grafana Adaptor
```

Comment:
Not quite right.  grafana.q doesn't require a separate file. These shoudl be added to config/settings/default.q